Form/Type/MapType.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Netgen\Bundle\EzFormsBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Class MapType
+ * @package Netgen\Bundle\EzFormsBundle\Form\Type
+ */
+class MapType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add("address", "text", array(
```

Comment:
Mislim da za to nema potrebe buduci da se svako polje ionak prefixa sa nazivom forme, pa nakon toga sa nazivom podforme. Tako da konacni naziv polja npr. u slucaju update-a, je ezforms_update_content_gmap_location_address.